components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -462,6 +463,19 @@ - (UIView *)viewForItem:(UITabBarItem *)item {
   return _itemViews[itemIndex];
 }
 
+- (UITabBarItem *)tabBarItemForPoint:(CGPoint)point {
+  for (NSUInteger i = 0; i < self.itemViews.count; i++) {
```

Comment:
Consider also checking `i < self.items.count` to minimize the number of frame computations below, if these values can get out of sync.

Reply:
Suggestion:

```objc
for (NSUInteger i = 0; (i < self.itemViews.count) && (i < self.items.count); i++) {
```

Reply:
Good catch.  Done!